### PR TITLE
Add: Adding sameSite cookie attribute for rb bindings

### PR DESCRIPTION
**Pull request by harsha509**
### Description
Adding sameSite cookie attribute for rb bindings and added tests

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: rb/lib/selenium/webdriver/common/manager.rb [hunk 1]
```diff
@@ -47,7 +47,7 @@ def add_cookie(opts = {})
 
         opts[:path] ||= '/'
         opts[:secure] ||= false
-
+        opts[:sameSite]
```
Comment (p0deje) [inline rb/lib/selenium/webdriver/common/manager.rb]: We don't need this line, do we?

File: rb/spec/integration/selenium/webdriver/manager_spec.rb [hunk 2]
```diff
@@ -64,6 +64,19 @@ module WebDriver
           expect(cookies.first[:value]).to eq('bar')
         end
 
+        it 'should add sameSite cookie with attribute Strict', only: {browser: :chrome} do
+          driver.navigate.to url_for('xhtmlTest.html')
+          driver.manage.add_cookie name: 'foo', value: 'bar', sameSite: 'Strict'
```
Comment (p0deje) [inline rb/spec/integration/selenium/webdriver/manager_spec.rb]: I believe we should use snake case for paramater to make the code more Rubyish:

```suggestion
          driver.manage.add_cookie name: 'foo', value: 'bar', same_site: 'Strict'
```

File: rb/spec/integration/selenium/webdriver/manager_spec.rb [hunk 3]
```diff
@@ -64,6 +64,19 @@ module WebDriver
...
+
+          expect(driver.manage.cookie_named('foo')[:sameSite]).to eq('Strict')
+        end
+
+        it 'should add sameSite cookie with attribute Lax', only: {browser: :chrome} do
+          driver.navigate.to url_for('xhtmlTest.html')
+          driver.manage.add_cookie name: 'foo', value: 'bar', sameSite: 'Lax'
+          expect(driver.manage.cookie_named('foo')[:sameSite]).to eq('Lax')
```
Comment (p0deje) [inline rb/spec/integration/selenium/webdriver/manager_spec.rb]: Likewise, I believe we need to use snake case 
```suggestion
          expect(driver.manage.cookie_named('foo')[:same_site]).to eq('Lax')
```

File: rb/lib/selenium/webdriver/common/manager.rb [hunk 4]
```diff
@@ -47,7 +47,7 @@ def add_cookie(opts = {})
```
Comment (p0deje) [inline rb/lib/selenium/webdriver/common/manager.rb]: Can you update the documentation of the `add_cookie` method to include a new parameter and mention it's Chrome-only?

File: rb/lib/selenium/webdriver/common/manager.rb (hunk 1 above)
Comment (Harsha509) [reply rb/lib/selenium/webdriver/common/manager.rb]: Hi @p0deje ,

Thanks for the review. No, it's not required feature works removing this. 
Updating the code

File: rb/lib/selenium/webdriver/common/manager.rb (hunk 4 above)
Comment (Harsha509) [reply rb/lib/selenium/webdriver/common/manager.rb]: Updated as `  # @option opts [String] :sameSite (Strict or Lax) currently supported only in chrome 80+ versions`

File: rb/spec/integration/selenium/webdriver/manager_spec.rb (hunk 3 above)
Comment (Harsha509) [reply rb/spec/integration/selenium/webdriver/manager_spec.rb]: Updated!

**Pull request merged**